src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -1097,10 +1097,14 @@ Copyright (c) .NET Foundation. All rights reserved.
       <UseSharedCompilation>true</UseSharedCompilation>
     </PropertyGroup>
 
+    <ItemGroup>
+      <SatelliteAssemblyReferences Include="@(ReferencePath)" Condition="'%(Filename)' == 'mscorlib' or '%(Filename)' == 'netstandard' or '%(Filename)' == 'System.Runtime' " />
```

Comment:
Given this is defined and used in the same file, should it be `_SatteliteAssemblyReferences` to indicate it's a _private_ variable? Or is the intent that outside code can depend on this?

Reply:
It should definitely be private, I'm just hacking around here.